python/lsst/display/firefly/firefly.py

```diff
@@ -140,7 +147,8 @@ def _mtv(self, image, mask=None, wcs=None, title=""):
                 self._fireflyFitsID = _fireflyClient.upload_data(fd, 'FITS')
 
             ret = _fireflyClient.show_fits(self._fireflyFitsID, plot_id=str(self.display.frame),
-                                           Title=title, MultiImageIdx=0)
+                                           Title=title, MultiImageIdx=0,
+                                           PredefinedOverlayIds=' ')
```

Comment:
If this is really supposed to be a single space, rather than an empty string or some other more typical custom value, it probably merits a code comment explaining why.

Reply:
It does have to be a space, or else the Javascript side discards the parameter...I'll add a comment